### PR TITLE
103 admin login to admin dashboard

### LINKED ISSUE #103: Admin login direct to admin dashboard
- admin login does not currently direct us to the admin dashboard
- directs to user dashboard

### PR DESCRIPTION
**Pull request by meyerhoferc**
fixes admin redirect to /admin/dashboard.
This wasn't working because of the stubbing in the test. I added a separate authentication for admin in the same #admin_login method in Application Controller

all tests pass
closes #103

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -17,12 +17,15 @@ def current_admin?
...
-  end
-
   def define_categories
     @categories = Category.all
   end
 
+  def admin_login
+    user = User.find_by(username: params[:session][:username])
+    if user && user.authenticate(params[:session][:password]) && user.admin?
+      "Logged in as #{user.username}"
```
Comment (cmacaulay) [inline app/controllers/application_controller.rb:27]: is this a flash message?

File: app/controllers/sessions_controller.rb [hunk 2]
```diff
@@ -8,7 +8,7 @@ def create
       flash[:notice] = "Logged in as #{user.username}"
       redirect_to dashboard_path
     else
-      flash.now[:notice] = "Your email or password"
+      flash.now[:notice] = "Your username or password is incorrect"
```
Comment (cmacaulay) [inline app/controllers/sessions_controller.rb:11]: 👍

**Review by cmacaulay — APPROVED** [review]
👍 Looks good!

File: app/controllers/sessions_controller.rb (hunk 2 above)
Comment (valerietrudell) [inline app/controllers/sessions_controller.rb]: 👍

**Comment by valerietrudell** [comment]
Looks good!

**Pull request merged**